app/assets/javascripts/discourse/tests/unit/lib/key-value-store-test.js

```diff
@@ -2,18 +2,37 @@ import { module, test } from "qunit";
 import KeyValueStore from "discourse/lib/key-value-store";
 
 module("Unit | Utility | key-value-store", function () {
-  test("it's able to get the result back from the store", function (assert) {
-    const store = new KeyValueStore("_test");
+  test("is able to get the result back from the store", function (assert) {
+    const store = new KeyValueStore("test_");
```

Comment:
I'm curious, why the change from `_test` to `test_`

Reply:
The `context` argument acts as a prefix to saved values. In the actual app the contexts are `discourse_`, `discourse_desktop_notifications_`, `discourse_push_notifications_`, `discourse_emojis_`. Makes sense for tests to reflect the actual usage. 😃